docker/lr-papermill-base/Makefile

```diff
@@ -0,0 +1,12 @@
+VERSION = 2.3.4
```

Comment:
can we update this to the latest stable (2.5.0)?
my bad for delaying the review so much that you'd need to update this.